### DISCUSSION TITLE
Multi-tenant example - can't log in as "user"

### DISCUSSION THREAD

**Discussion by nhwebforge** [payloadcms/payload#6594] [General]
Hello!

So, I've local-installed the multi-tenant example and it's up and running. Connecting happily to my MongoDB. I can log in as super-admin on localhost:3000/admin no problem. Changed hosts file to reflect the ".com" and I can log in as the "abc" admin on abc.localhost.com:3000 and the "bbc" admin bbc.localhost.com:3000.  So far, so good.

However, when I log in as "user@bbc.com" or "user@abc.com" all I get is "Unauthorized, you must be logged in to make this request. You are not allowed to access this page." and am invited to log in.

What exactly am I supposed to see when I log in as a simple "user"? How can I view the "abc-home" or "bbc-home" pages that are configured? Should it be as simple as going to abc.localhost.com:3000/abc-home?

Thanks in advance for any advice you can provide.

NH

**Comment by JarrodMFlesch** [comment]
The user accounts are acting as users that can interact with the payload API endpoints but cannot access the admin panel. This is common if you have an app that users can log into but you do not want them to log into the payload dashboard. Think like, a mobile app, you have users that log into the app(downloaded from the App Store) but not to the app that manages the content in the app (payload).

The same could apply to a web app, your app might have content that needs managed but you have a separate app built out for users to use – specific to your app.

**Reply by nhwebforge** [reply]
Thanks Jarrod, at least that confirms my predictions and it's exactly what I'm looking for. However, how do I view the "abc-home" and "bbc-home" pages? What should the URL be? I see the slug but it doesn't want to work with abc.localhost.com:3000/abc-home.  I presume a "user" should at the very least be able to view those pages.

I'm working on a web app where a user signs up, is allocated a tenant, and they can then invite other users to that tenant as "users". There are many complicated and high faluted examples of multi-tenant architectures, but I'm hoping Payload will be a straight-forward solution. The planned app is basically just a CRUD app but siloed by tenant.

Really appreciate your assistance!

**Reply by roybarber** [reply]
@nhwebforge Did you manage to resolve this?
